packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -176,6 +176,11 @@ export interface IContextualMenuProps extends React.Props<ContextualMenu> {
    */
   calloutProps?: ICalloutProps;
 
+  /**
+   * the total size of the set
```

Comment:
While these could be useful, they should not have been included in this change... they are artifacts from when I was playing around with pushing the swatch color picker through contextualMenu.

No one is using those props right now and can always be added later if needed. I've removed them from this PR